pyensign/events.py

```diff
@@ -321,6 +323,55 @@ class EventState(Enum):
     NACKED = 4
 
 
+def from_object(obj, mimetype=None, encoder=None):
+    """
+    Makes a best effort attempt to convert a Python object into an Event. If an encoder is provided, it will be used to encode the object to bytes. If no encoder
+    is provided, the object will be encoded according to its type or mimetype.
```

Comment:
This allows arbitrary Python objects to be converted into Ensign events. I'm not sure if this makes sense yet...